ground/src/main/java/com/google/android/ground/ui/home/HomeScreenViewModel.kt

```diff
@@ -97,23 +117,41 @@ internal constructor(
     navigator.navigate(HomeScreenFragmentDirections.actionHomeScreenFragmentToSettingsActivity())
   }
 
-  fun onLocationOfInterestClick(mapLocationsOfInterest: ImmutableList<MapLocationOfInterest>) {
+  fun onLocationOfInterestClick(features: ImmutableList<Feature>) {
```

Comment:
Just by the name, this method seems very similar to `onMarkerClick`. Is it possible to merge the two?

Reply:
yes, I think merging these is a good idea -- let's bump it to a separate PR so that it's easier to validate behaviors. I'll open a follow-up after this one.